extension/content.js

```diff
@@ -128,7 +128,7 @@ function addDeleteForkLink() {
 
 	if ($postMergeContainer.length > 0) {
 		const $postMergeDescription = $postMergeContainer.find('.merge-branch-description');
-		const forkPath = $postMergeContainer.attr('data-channel').split(':')[0];
+		const forkPath = `${getUsername()}/${repoName}`;
```

Comment:
Thanks, I had a feeling you were thinking this, and really we need the "source" to make the URL. Maybe we can do it from the linkified branches?